setup.cfg

```diff
@@ -1,3 +1,86 @@
+[metadata]
+name = pylint
+license = GPL-2.0-or-later
+license_file = COPYING
+description = python code static checker
+keywords = static code analysis linter python lint
+long_description = file: README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+author = Python Code Quality Authority
+author_email = code-quality@python.org
+url = https://github.com/PyCQA/pylint
+project_urls =
+    What's New=https://pylint.pycqa.org/en/latest/whatsnew/
```

Comment:
Although not technically needed, can you add quotes here? That would improve the readability and enable better syntax highlighting.
```ini
project_urls =
    "What's New" = https://....
```